airbyte-server/src/test/java/io/airbyte/server/migration/DatabaseArchiver.java

```diff
@@ -48,6 +48,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+// todo (cgardens) - kill this class. only kept around because it is hard to extricate from the
+// {@link RunMigrationTest} test.
```

Comment:
see comment in description of PR. I want to kill this class but for now it's too hard to pull out of a test to make it worthwhile.